src/gui_common/MainMenu.cs

```diff
@@ -51,6 +51,7 @@ public class MainMenu : Node
 
     public override void _Ready()
     {
+        CallMiscTranslations();
```

Comment:
This doesn't need to be actually ever be called, if there is a warning just put suppression on the method and add a documentation on it why the method exists.